portfolio/target/appengine-staging/WEB-INF/classes/source-context.json

```diff
@@ -0,0 +1 @@
+{"git": {"url": "https:\/\/github.com\/bhassan6621\/StepTraining.git", "revisionId": "6f5c9dc873b3cc4f9a5aac3e699c0776455109dd"}}
```

Comment:
add new line

Reply:
I cant seem to find these files inside of the target folder